blockchain/filedao/filedao_v2_util.go

```diff
@@ -215,12 +200,26 @@ func (fd *fileDAOv2) getReceipt(height uint64) ([]*action.Receipt, error) {
 	return fd.deser.ReceiptsFromBlockStoreProto(blockStore)
 }
 
+func (fd *fileDAOv2) getFromStagingBuffer(height uint64) (*block.Store, error) {
+	if fd.loadTip().Height-height >= fd.header.BlockStoreSize {
+		return nil, ErrNotSupported
+	}
```

Comment:
staging buffer contains the top `BlockStoreSize` blocks, for example, if tip height = 100, it has blocks 100, 99, ... 85

Reply:
The return value of `blkStore.Size()` is currently incorrect, and it seems that this PR did not fix it. Shouldn't the Size method be removed? If Size is used elsewhere, it should also be checked for similar issues.

Reply:
will have another PR